manifests/init.pp

```diff
@@ -241,6 +250,9 @@
   Boolean $cluster_enabled       = $::redis::params::cluster_enabled,
   $cluster_config_file           = $::redis::params::cluster_config_file,
   $cluster_node_timeout          = $::redis::params::cluster_node_timeout,
+  Integer $cluster_slave_validity_factor = $::redis::params::cluster_slave_validity_factor,
```

Comment:
Please align the `=` for all parameters. Can you also define the lowest/highlest allowed value for the integer parameters? At least the lowest value should be enforced. This is probably 0 or 1?

Reply:
Done.
Both of them have minimum from 0, I have set cluster_migration_barrier default to 1 as per redis docs